test/libethereum/blockchain.cpp

```diff
@@ -36,86 +38,45 @@ using namespace dev::eth;
 
 namespace dev {  namespace test {
 
-typedef std::vector<bytes> uncleList;
-typedef std::pair<bytes, uncleList> blockSet;
-
-using BlockHeader = Ethash::BlockHeader;
-
-BlockHeader constructBlock(mObject& _o, h256 const& _stateRoot = h256{});
-bytes createBlockRLPFromFields(mObject& _tObj, h256 const&  _stateRoot = h256{});
-RLPStream createFullBlockFromHeader(BlockHeader const& _bi, bytes const& _txs = RLPEmptyList, bytes const& _uncles = RLPEmptyList);
-
-mArray writeTransactionsToJson(Transactions const& _txs);
-mObject writeBlockHeaderToJson(mObject& _o, BlockHeader const& _bi);
-void overwriteBlockHeader(BlockHeader& _current_BlockHeader, mObject& _blObj, const BlockHeader& _parent);
-void updatePoW(BlockHeader& _bi);
-mArray importUncles(mObject const& _blObj, vector<BlockHeader>& _vBiUncles, vector<BlockHeader> const& _vBiBlocks, std::vector<blockSet> _blockSet);
+//Functions that working with test json
+void compareBlocks(TestBlock const& _a, TestBlock const& _b);
+mArray writeTransactionsToJson(TransactionQueue const& _txsQueue);
+mObject writeBlockHeaderToJson(Ethash::BlockHeader const& _bi);
+void overwriteBlockHeaderForTest(mObject const& _blObj, TestBlock& _block, std::vector<TestBlock> const& importedBlocks);
+void overwriteUncleHeaderForTest(mObject& _uncleHeaderObj, TestBlock& _uncle, std::vector<TestBlock> const& _uncles, std::vector<TestBlock> const& _importedBlocks);
```

Comment:
since you are `using namespace std` you don't need that in this file (at several places)